.changeset/small-cycles-fold.md

```diff
@@ -2,4 +2,4 @@
 "@salt-ds/lab": patch
 ---
 
-ComboBoxNext: Fix missing forwardedRef on `DefaultListItem`
+Fixed Combo Box Next missing not applying a ref to its `DefaultListItem`.
```

Comment:
this sounds a little weird.. was it meant to be missing `forwardedRef` ?

Reply:
Thanks there's a grammar mistake here